content/brazilian-portuguese/arduino/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Arduino: Jogo de luz"
+description: "Construa um microcontrolador Arduino"
+date: 2020-07-27T14:04:59.61-07:00
+difficulty: "Intermediário"
+icon: "fas fa-microchip"
+draft: false
+---
+
+## Introdução - Construindo com Arduino
+
+Eletrônicos como microcontroladores, sensores e todos os tipos de circuitos estão ao nosso redor! Alguns são muito básicos e fornecem informações de telemetria para sistemas de tráfego. Outros executam os semáforos programados com temporização embutida.
+
+Hoje trabalharemos em algumas atividades para mostrar algumas das capacidades desta plataforma. Começaremos pequenos e simples e construiremos para maiores e mais complexos. Você aprenderá todas as informações básicas sobre como usar a placa controladora Arduino, sensores e componentes. Depois de fazer cada atividade, sinta-se à vontade para mudar algumas coisas para ver o que é diferente e obter uma compreensão mais profunda
```

Comment:
"mudar algumas coisas para ver o que é diferente" doesn't fit it well. Suggestion: "mudar alguns parâmetros para ver o que muda"